app/createTournament/createTournamentBusiness.js

```diff
@@ -0,0 +1,53 @@
+import * as common from '../../utils/common';
+import * as playerValidation from '../playersinscription/addPlayerBusiness';
+import {Player} from '../Player';
+import * as Math from '../../node_modules/mathjs/dist/math.min';
+
+export function validateNumberofPlayers(){
+
+var numberofPlayers = $('#tourneyPlayersList  >tbody >tr').length;
```

Comment:
Don't use DOM as a data storage. Use JS objects instead. I'd expect an array of players which will render to HTML, but not the other way around (HTML -> domain objects)